Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -445,6 +445,40 @@ + (BOOL)date:(NSDate*)date isBetweenDate:(NSDate*)beginDate andDate:(NSDate*)end
     return YES;
 }
 
+- (BPKSimpleDate *)simpleDateFromDate:(NSDate *)date {
+    if(date == nil) {
```

Comment:
Can we explicitly mark the return type and parameter as `nullable` or `nonnull` and either remove this check or keep it.